app/src/main/java/com/example/lhi06/residue/FindAlbumActivity.java

```diff
@@ -22,12 +24,14 @@
 
 public class FindAlbumActivity extends AppCompatActivity {
 
+    public final static String ALBUM_EXTRA = "com.example.lhi06.residue.ALBUM_EXTRA";
+
     private AlbumAdapter adapter;
     private ItunesService service;
 
 
-    @BindView(R.id.recycleView) RecyclerView recyclerView;
-    @BindView(R.id.request) EditText editText;
+    @BindView(R.id.recycleview_albums_findalbum) RecyclerView recyclerView;
```

Comment:
I would rename the id to just `recyclerview_albums`. No need to specify that it is part of the find albums activity